src/main/resources/com/google/api/codegen/nodejs/package.json.snip

```diff
@@ -25,23 +29,38 @@
       "{@metadata.shortName}",
       "{@metadata.fullName}"
     ],
+    "scripts": {
+      "cover": "nyc --reporter=lcov mocha --require intelli-espower-loader test/*.js && nyc report",
+      "docs": "repo-tools exec -- jsdoc -c .jsdoc.js",
+      "generate-scaffolding": "repo-tools generate all && repo-tools generate lib_samples_readme -l samples/ --config ../.cloud-repo-tools.json",
+      "lint": "repo-tools lint --cmd eslint -- src/ samples/ system-test/ test/",
+      "prettier": "repo-tools exec -- prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js",
+      "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
+      "system-test": "repo-tools test run --cmd mocha -- system-test/*.js --no-timeouts",
+      "test-no-cover": "repo-tools test run --cmd mocha -- test/*.js --no-timeouts",
+      "test": "repo-tools test run --cmd npm -- run cover"
+    },
     "dependencies": {
       @join packageDep : metadata.dependencies on ",".add(BREAK)
         "{@packageDep.name}": "^{@packageDep.versionBound.lower}"
       @end
     },
     "devDependencies": {
-      "mocha": "^3.2.0",
+      "@@google-cloud/nodejs-repo-tools": "^2.1.0",
+      "async": "^2.5.0",
+      "codecov": "^3.0.0",
+      "eslint": "^4.9.0",
```

Comment:
These are not really properties of the package itself. It would be valid to output this tool and use `semistandard` instead of `prettier`, for example. Can we have repo-tools add this sort of thing, so we put out a generic package that any third party could run through their preferred tooling?